packages/markdown/src/vaadin-markdown.js

```diff
@@ -43,10 +44,33 @@ class Markdown extends ElementMixin(ThemableMixin(PolylitMixin(LitElement))) {
     `;
   }
 
+  static get properties() {
+    return {
+      /**
+       * The Markdown content.
+       *
+       * @type {string}
+       */
+      markdown: {
+        type: String,
```

Comment:
Should this use `sync: true` for consistency with other components?

Reply:
Aren't we mostly using that for the Polymer -> Lit migration. I don't see `sync` use in the components that have been originally built with LitElement (side-nav, dashboard, card)

Reply:
I suppose it could be problematic with the Flow counterpart if setting `content` doesn't immediately update the content (so that we'd have to wait for it to be updated in ITs)?

Reply:
Could be, doesn't harm making it sync also. Done.